main.tf

```diff
@@ -456,6 +456,26 @@ resource "null_resource" "confirm_network_healthy" {
   }
 }
 
+##############################################################################
+# OCP Console Patch enablement
+##############################################################################
+resource "null_resource" "ocp_console_management" {
+
+  count      = var.verify_worker_network_readiness ? 1 : 0
```

Comment:
why are you using `verify_worker_network_readiness` in the count? That has nothing to do with enabling or disabling the OCP console

Reply:
My understanding was that once the cluster is fully up and connected then ocp console should be applied but I get it now that it is unrelated. I have removed it. Thank you.